documentation/src/main/asciidoc/ch11.asciidoc

```diff
@@ -384,3 +384,11 @@ dependency injection in constraint validators.
 Hibernate Validator also provides support for the unwrapping of JavaFX properties. If JavaFX is present
 on the classpath, ``ValueExtractor``s for JavaFX properties are automatically registered.
 See <<section-valueextraction-javafx>> for examples and further discussion.
+
+
+[[validator-osgi]]
+=== OSGi
+
+Starting with Hibernate Validator 9.0, the Hibernate team no longer tests nor maintains the OSGi integration.
+For users who would want to experiment with Hibernate Validator in an OSGi environment,
+we will keep the existing manifest entries in place as they were.
```

Comment:
Hey @yrodiere 
I guess this should be enough in the guide to state the state of things related to OSGi? 🙈 

And then in the release post / migration guide / on the site for the 9.0 series I'm thinking we can also add something like:

> Even though we no longer maintain the Hibernate Validator OSGi integration, we do encourage the community members interested in it to come forward with pull requests if they encounter some issues. In such a case, we will try to guide the submitter through the contribution process and assess the suggested patch for inclusion in the build.